modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/services/SeqexecWebClient.scala

```diff
@@ -77,6 +78,17 @@ object SeqexecWebClient extends ModelBooPicklers {
   }
 
   /**
+    * Requests the backend to set the Conditions globally
+    */
+  def setConditions(conditions: Conditions): Future[RegularCommand] = {
+    Ajax.post(
+      url = s"$baseUrl/commands/conditions",
```

Comment:
I'm not sure what it takes to be a `command`. This is telling the engine to set/change the conditions. Which criteria should follow to make something a `command`?